packages/e2e/package.json

```diff
@@ -5,6 +5,7 @@
   "description": "End to end tests for the cardano-js-sdk packages.",
   "main": "index.js",
   "scripts": {
+    "artillery:pool": "WORKERS=1 node -r ts-node/register ../../node_modules/.bin/artillery run artillery/StakePoolSearch.yml",
```

Comment:
:sunglasses: We may also need to compile this to run distributed tests, but it's a nice solution for the current scope.

Reply:
When we need to compile, no problem: we are free to use a `js` file, it is supported.

The _only_ reason for this is for development: while writing a new test, the developer don't need to build the SDK **at each** run attempt of its test.

This will save a lot of team time while writing artillery tests.